ksql-engine/src/test/resources/query-validation-tests/joins.json

```diff
@@ -9,7 +9,167 @@
   ],
   "tests": [
     {
-      "name": "left join",
+      "name": "stream stream left join",
+      "statements": [
+        "CREATE STREAM TEST (ID bigint, NAME varchar, VALUE bigint) WITH (kafka_topic='left_topic', value_format='DELIMITED', key='ID');",
+        "CREATE STREAM TEST_STREAM (ID bigint, F1 varchar, F2 bigint) WITH (kafka_topic='right_topic', value_format='DELIMITED', key='ID');",
+        "CREATE STREAM LEFT_OUTER_JOIN as SELECT t.id, name, value, f1, f2 FROM test t left join TEST_STREAM tt on t.id = tt.id SPAN 11 seconds;"
```

Comment:
Should we also add another test that has a SPAN with BEFORE and AFTER set?

Reply:
Yes. I intend to do this.

Reply:
This has been added.